AutoRest/Generators/Extensions/Extensions/Extensions.cs

```diff
@@ -410,13 +410,18 @@ public static void AddParameterGroups(ServiceClient serviceClient)
                         parameterGroupType = new CompositeType
                         {
                             Name = parameterGroupName,
-                            Documentation = "Additional parameters for the " + method.Name + " operation."
+                            Documentation = "Additional parameters for these operations: " + method.Name
```

Comment:
The list of operations could be potentially very long. In the limit, it could be every operation supported by the client. Do we really need to list them in the comment?